corehq/apps/app_manager/tests/data/v2_diffs/templates/form_view.html.diff.txt

```diff
@@ -7,48 +7,24 @@
  {% load hq_shared_tags %}
  {% load i18n %}
  
--    {% include "app_manager/v1/partials/xpathValidator.html" %}
--    {% include "app_manager/v1/partials/nav_menu_media_js_common.html" %}
--    {% include "app_manager/v1/partials/nav_menu_media_js.html" with item=multimedia.menu %}
-+    {% include "app_manager/v2/partials/xpathValidator.html" %}
-+    {% include "app_manager/v2/partials/nav_menu_media_js_common.html" %}
-+    {% include "app_manager/v2/partials/nav_menu_media_js.html" with item=multimedia.menu %}
-     {% if allow_form_workflow %}
-         <script src="{% static 'app_manager/js/form_workflow.js' %}"></script>
-     {% endif %}
-@@ -174,15 +174,18 @@
-             });
-         }
- 
--        $('#form-filter').koApplyBindings(new FormFilter());
+ {% block js-inline %}{{ block.super }}
 -
-+
-+        var $formFilter = $('#form-filter');
-+        if ($formFilter.length) {
-+            $formFilter.koApplyBindings(new FormFilter());
-+        };
-         {% endif %}
- 
-         {% if allow_form_workflow %}
-             var FormWorkflow = hqImport('app_manager/js/form_workflow.js').FormWorkflow;
-             var labels = {};
-             labels[FormWorkflow.Values.DEFAULT] = "{% trans "Home Screen" %}";
--            labels[FormWorkflow.Values.ROOT] = "{% trans "Module Menu" %}";
-+            labels[FormWorkflow.Values.ROOT] = "{% trans "Main Menu" %}";
-             {% with module_name=module.name|trans:langs %}
-             labels[FormWorkflow.Values.MODULE] = "{% trans "Module:" %} {{ module_name|escapejs }}";
-             {% endwith %}
-@@ -225,7 +228,7 @@
-         });
-         {% endif %}
- 
--        {% if allow_cloudcare %}
-+        {% if can_preview_form and allow_cloudcare %}
-             // tag the 'preview in cloudcare' button with the right url
-             // unfortunately, has to be done in javascript
-             var getCloudCareUrl = function(urlRoot, appId, moduleId, formId, caseId) {
-@@ -291,75 +294,58 @@
-     </script>
+ {% if request.guided_tour %}{% include request.guided_tour.template %}{% endif %}
```

Comment:
we handle which guided tour is provided on the server side: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/forms.py#L655-L656